src/services/uploadService.ts

```diff
@@ -131,6 +149,7 @@ class UploadService {
             try {
                 await this.fetchUploadURLs(token);
             } catch (e) {
+                console.log('error fetching uploadURLs', e);
```

Comment:
`console.error(...)`.

Reply:
Will do this in the sentry integration PR